lib/wal/wal.c

```diff
@@ -90,6 +91,66 @@ struct wal_sync_waiter {
 void
 wal_ionotify_cb(void *cbarg, merr_t err);
 
+HSE_ALWAYS_INLINE static void
+wal_throttle_sensor_set(struct wal *wal, uint64_t bufsz, uint64_t buflen)
```

Comment:
What is the description of this function? Set the parameters of the wal throttle sensor?

Reply:
Done, thnx!